pkg/repository/stock/stockRepositoryImpl.go

```diff
@@ -81,7 +88,10 @@ func (r *stockRepositoryImpl) Update(ctx context.Context, stockid string, stock
 			return nil, errors.New(fmt.Sprintf("failed to update stock"))
 		}
 	}
-	return stocks.ToStockEntity(), nil
+
+	stockEntity := stocks.ToStockEntity()
+	r.SetStockSubAttributes(stockEntity, sp)
+	return stockEntity, nil
 }
 
 func (r *stockRepositoryImpl) Delete(ctx context.Context, id string) (*entities.Stock, error) {
```

Comment:
Why it **not** have DB connection? in Delete Function

Reply:
it have but i not fix before commit, it not show